--- libtool	2015-10-13 10:55:30.000000000 +0200
+++ libtool2	2015-10-13 10:59:35.000000000 +0200
@@ -10724,7 +10724,7 @@
 	# But is supposedly fixed on 10.4 or later (yay!).
 	if test CXX = "$tagname"; then
 	  case ${MACOSX_DEPLOYMENT_TARGET-10.0} in
-	    10.[0123])
+        10.[0123][,.])
 	      func_append compile_command " $wl-bind_at_load"
 	      func_append finalize_command " $wl-bind_at_load"
 	    ;;
